std_msgs/Header header
uint64 robot_id
uint64 target_id
bool lost
float64 x
float64 y
float64 vx
float64 vy